planning/behavior_path_planner/autoware_behavior_path_planner/include/autoware/behavior_path_planner/planner_manager.hpp

```diff
@@ -575,6 +575,10 @@ class PlannerManager
   ModuleUpdateInfo debug_info_;
 
   std::shared_ptr<SceneModuleVisitor> debug_msg_ptr_;
+
+  mutable BehaviorModuleOutput last_valid_reference_path_;
+
+  mutable bool has_last_valid_reference_path_;
```

Comment:
Please use `optional<BehaviorModuleOujtput> last_valid_rerference_path_` and set a valid value if and only if the reference path was not empty.

Reply:
Thank you, fixed in [3fb679c](https://github.com/autowarefoundation/autoware.universe/pull/10205/commits/3fb679cd12d3dbad7c8e5171cfaea636a7b202a7)

Reply:
@soblin Could you check my fix?

Reply:
@soblin Could you check my fix?